src/contracts/BlockStorage.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity 0.8.11;
+
+
+contract BlockStorage {
```

Comment:
This is extra. Create an interface for the `SpectreProxy` contract and all other state root storage contracts.
`IStateRootStorage`